frontend/cs/r1cs/api_assertions.go

```diff
@@ -83,14 +83,12 @@ func (builder *builder) AssertIsBoolean(i1 frontend.Variable) {
 func (builder *builder) AssertIsLessOrEqual(_v frontend.Variable, bound frontend.Variable) {
 	v := builder.toVariable(_v)
 
-	switch b := bound.(type) {
-	case expr.LinearExpression:
+	if b, ok := bound.(expr.LinearExpression); ok {
```

Comment:
Is it only stylistic change?

Reply:
yes -- dirty history, my bad. (one attempt I add `api.MAC` take a pointer as a parameter to grow the linear expression, but it complicated things in too many places)